docs/readme.md

```diff
@@ -1,4 +1,4 @@
-# Documentation Governance
+# CivicActions Documentation 
```

Comment:
I have changed "Documentation" to "Handbook" if that was confusing.

I, personally, am confused between what goes into the CONTRIBUTING.md file versus the readme of the "docs" folder (because i'm advocating that each folder should have a readme.md).

Maybe we just remove "docs" folder altogether.